menuconfig.py

```diff
@@ -299,6 +299,141 @@
     "brightpurple": curses.COLOR_MAGENTA + 8,
 }
 
+def rgb_to_6cube(rgb):
+    """
+    Convert each component of a RGB color to its approximate index
+    in the xterm 6cube color palette
+    """
+    # The formula for converting a single RGB value to the 6cube index
+    rto6 = lambda x: 0 if x < 48 else max(1, int(round((x - 55) / 40)))
```

Comment:
Would be nice to either explain the formula or include a link to some reference here.

Can just say "Converts an RGB value to a 6cube index. (Explanation ...)" too.